html/pfappserver/lib/pfappserver/Util/ACLs.pm

```diff
@@ -20,6 +20,9 @@ sub _validate_acl {
     my ($field) = @_;
     my $acl = $field->value;
     if ($acl) {
```

Comment:
We should also test the acl limit of this switch, like if it's support acls/dlacls and it reach the limit then trigger an error.